shared/utils/getTasks.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+const TICKED_CHECKBOX_REGEX = /\[(x)\]\s/g;
+const CHECKBOX_REGEX = /\[(x|\s|)\]\s/g;
```

Comment:
Could we use this regex to match what the editor uses?

https://github.com/outline/rich-markdown-editor/blob/main/src/lib/markdown/checkboxes.ts#L4

That way we can run the regex only once and then look at the contents of the first capture group to determine if the checkbox is checked or not

Reply:
I was conscious of the head ups, but I could not understand the editor's regex.

Is it possible to have `-` and `_` inside the brackets? Also, it has capital X instead of x, whereas the document `text` attribute has `x` inside the brackets.

Reply:
It's possible although granted unlikely in imported markdown. Just thinking for consistency's sake if they appear as checkboxes in the editor they should _always_ show up in the tasks list too.

The capitalization doesn't matter when the case insensitive flag is used